public/js/report/controller.js

```diff
@@ -278,6 +279,9 @@ angular.module('app').controller('reportCtrl', function ($scope, connection, $co
             $scope.time = result.time;
 
             $scope.$broadcast('repaint', { fetchData: false, data: result.data });
+        }).catch(err => {
+            $scope.$broadcast('stopLoading');
+            notify.error(err.message);
```

Comment:
The whole response body is displayed, which is not very useful, and can be very long or empty.
We should display a human-readable (and translatable) message instead, and optionally log the message in the console

Reply:
I replaced err.message with a more generic message and for more details I added a console.error statement with the complete error